### DISCUSSION TITLE
timeout error

### DISCUSSION THREAD

**Discussion by yugdobariya** [soranoo/TradingView-Free-Webhook-Alerts#52] [General]
bro its showing this error please help me with it.
TIMEOUT: TIMEOUT
    at Timeout._onTimeout (file:///var/task/lambda_handler.mjs:889:23)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
![Screenshot (5524)](https://github.com/soranoo/TradingView-Free-Webhook-Alerts/assets/60605739/cdcc1fca-c4ae-4468-87e5-af21ad060e5c)

**Comment by yugdobariya** [comment]
also its not responding in my bybit testnet 
my message is this:
{"symbol":"{{ticker}}","side":"{{strategy.order.action}}","qty":"{{strategy.order.contracts}}","price":"{{close}}","trigger_time":"{{timenow}}","strategy_id":"c11075e6-e5ee-4032-8457-c5596d8bd4d6"}

![Screenshot (5525)](https://github.com/soranoo/TradingView-Free-Webhook-Alerts/assets/60605739/17913a2f-504e-4eec-913f-6f9e2d4846b3)

**Comment by soranoo** [comment]
Any log you got after clicking the "Test" button before deploying?
